Dnn.CommunityForums/class/Utilities.cs

```diff
@@ -43,51 +43,6 @@ public abstract partial class Utilities
         internal static CultureInfo DateTimeStringCultureInfo = new CultureInfo("en-US", true);
 
 
-        /// <summary>
-        /// Calculates a friendly display string based on an input timespan
-        /// </summary>
-        public static string HumanFriendlyDate(DateTime displayDate, int ModuleId, int timeZoneOffset)
```

Comment:
This is removed... Is there a replacement?  Since it's public, shouldn't we follow a deprecation policy?

Reply:
The aren't removed altogether---just in that code file. I moved them to a different code file Deprecated/Utilities.cs, and marked them as deprecated in future release.